pkg/apis/kudo/v1beta1/operatorversion_types.go

```diff
@@ -210,6 +210,9 @@ type KudoOperatorTaskSpec struct {
 	// a specific operator version in the official repo, defaults to the most recent one
 	// +optional
 	OperatorVersion string `json:"operatorVersion,omitempty"`
+	// a parameter file name that will be used to populate Instance.Spec.Parameters
```

Comment:
The more I think about this.. the more I don't like this entry... I don't understand why we would have this in a CRD

Reply:
it may also be worth noting that the existing `kudo install -P` use of param files is an array...  I loss the battle of opinions that it should be simplified to one file.

Reply:
Well, it is as proposed in the KEP - and it is different from the CLIs `kudo install -P` although it servers a similar purpose. I agree that it is a bit misleading, I had to re-read the KEP as well to fully understand it

Reply:
Ken, we need such field in the operator task spec, this is because each operator task might use a different template in order to generate the parameter file for the child instance.

Reply:
@porridge I'm not sure I completely understand...  
I don't see a difference between storing a map of params or storing the file that needs to be parsed... is there a reason we can't parse the file and store the param map?

Reply:
regarding my last comment... I see.  I don't know if I like it... but I see it.
just because we have a configuration file for managing these concerns does mean we need to have an "file" as an entry in the CRDs...   The file needs to have some structure... we should read that structure and store that structure in the CRD IMO.

Reply:
@kensipe Aleksey did try designing it the way you suggest, but it ended up [being *very* complex and require extensive changes to the instance lifecycle](https://github.com/kudobuilder/kudo/pull/1454#commits-pushed-1e200e4). In the end we decided to pass parameter file templates instead.

Reply:
Regarding the name `parameterFile`: from the perspective of the operator developer, it is exactly what it is: a name of the file with parameters for the child operator. I'm open to suggestions though. 

As for why it needs to be a file and not an array is described in the KEP but the tl;dr is: the parameter file is templated and evaluated on the server-side, during plan execution (same as with parent `params.yaml` file)